pkgs/desktops/gnome-3/extensions/unite-shell/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, gnome3, fetchFromGitHub, xorg, glib, coreutils }:
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-extension-unite-shell";
+  version = "44";
+
+  src = fetchFromGitHub {
+    owner = "hardpixel";
+    repo = "unite-shell";
+    rev = "v${version}";
+    sha256 = "0nqc1q2yz4xa3fdfx45w6da1wijmdwzhdrch0mqwblgbpjr4fs9g";
+  };
+
+  uuid = "unite@hardpixel.eu";
+
+  nativeBuildInputs = [ glib ];
+
+  buildInputs = [ xorg.xprop ];
+
+  buildPhase = ''
+    runHook preBuild
+    glib-compile-schemas --strict --targetdir=${uuid}/schemas/ ${uuid}/schemas
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/share/gnome-shell/extensions
+    cp -r ${uuid} $out/share/gnome-shell/extensions
+    runHook postInstall
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Unite is a GNOME Shell extension which makes a few layout tweaks to the top panel and removes window decorations to make it look like Ubuntu Unity Shell.";
+    license = licenses.gpl3;
```

Comment:
gpl3 is deprecated, use gpl3Only or gpl3Plus if possible

Reply:
I can't change the license. I will open an issue so the owner might change it.

Reply:
Sure, it's not about changing the license, it's just about clarification. See https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206 for more details.

Reply:
Thank you for clearing that up. I assumed that it is a different license.